tests/unit/core/data/test_helpers.py

```diff
@@ -86,6 +86,7 @@ def generate_datumaro_dataset_item(
     task: str,
     image_shape: np.array = np.array((5, 5, 3)),
     mask_shape: np.array = np.array((5, 5)),
+    empty_label: bool = False,
```

Comment:
I think `empty_label` isn't used anywhere. Is it used somewhere I don't know?

Reply:
Thanks, I missed to delete it